### PR TITLE
feat(tier4_perception_launch): add arg to swtich lidar_centerpoint model

### PR DESCRIPTION
**Pull request by yukke42**
Signed-off-by: yukke42 <yusuke.muramatsu@tier4.jp>

## Description

This change helps to switch the lidar_centerpoint model when launching autoware.
e.g.: If you want to switch the model from `centerpoint_tiny` to `centerpoint`. (default is `centerpoint_tiny` now)

```bash
ros2 launch autoware_launch logging_simulator.launch.xml centerpoint_model_name:=centerpoint map_path:=...
```

e.g.: If you want to use the model in other directory s.t. our private model.
```bash
ros2 launch autoware_launch logging_simulator.launch.xml centerpoint_model_name:=MODEL_NAME \
    centerpoint_model_path:=PATH_TO_MODEL_DIRECTORY \ 
    centerpoint_model_param_path:=PATH_TO_PARAM_DIRECTORY \
    map_path:=... 
```

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -152,11 +155,16 @@
   <!-- CenterPoint -->
   <group if="$(eval &quot;'$(var lidar_detection_model)'=='centerpoint'&quot;)">
     <push-ros-namespace namespace="centerpoint"/>
-    <include file="$(find-pkg-share lidar_centerpoint)/launch/lidar_centerpoint.launch.xml">
-      <arg name="input/pointcloud" value="$(var input/pointcloud)"/>
-      <arg name="output/objects" value="objects"/>
-      <arg name="score_threshold" value="$(var score_threshold)"/>
-    </include>
+    <group>
```
Comment (miursh) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:158]: [Q] Why do you need group tag again? (Group tag is already set in L156)

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 2]
```diff
@@ -28,6 +28,9 @@
   <arg name="container_name" default="pointcloud_container"/>
   <arg name="use_validator" default="true" description="use obstacle_pointcloud based validator"/>
   <arg name="score_threshold" default="0.45"/>
+  <arg name="centerpoint_model_name" default="centerpoint_tiny" description="options: `centerpoint` or `centerpoint_tiny`"/>
```
Comment (miursh) [inline launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:31]: [Q] Is it still better to use centerpoint_tiny as default?

Comment (yukke42) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:31]: `centerpoint` is better, but this pr is aim to pass the model parameters from launch files to the lidar_centerpoint package.  So I will make another PR to change the default model at the same time as lidar_centerpoint package.

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 1 above)
Comment (yukke42) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:158]: Group tags are used in `lidar_based_detection.launch.xml` but not in `camera_lidar_based_detection.launch.xml` for centerpoint.
So I unified to use for https://github.com/autowarefoundation/autoware.universe/pull/1876

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 2 above)
Comment (miursh) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:31]: OK. Sounds good.

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 1 above)
Comment (miursh) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:158]: I don't think either of these are necessary... but it's ok

**Comment by yukke42** [comment]
I fixed the conflicts.

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**